src/themes/uswitch/theme.json

```diff
@@ -1525,6 +1525,11 @@
           "caret": {
             "variant": "compounds.accordion.base.caret"
           }
+        },
+        "titleIcon": {
```

Comment:
This can also go in the component since it's the same for all brands

Reply:
I put it in the theme to make it easier for other brands to change the icon sizes in the future if they need to